front_end/src/app/(campaigns-registration)/projM123/signup-and-register/page.tsx

```diff
@@ -1,8 +1,26 @@
+import ProfileApi from "@/services/profile";
+import { AlreadyRegisteredCard } from "../components/cards";
 import Header from "../components/header";
 import { RegistrationAndSignUpPage } from "../components/registration-forms";
 import { Heading1, Heading2 } from "../components/typography";
 
 export default async function Page() {
+  const user = await ProfileApi.getMyProfile();
+
+  if (user && user.registered_campaign_keys.indexOf("projM123") >= 0) {
```

Comment:
Out of curiosity, was there a specific reason why you used `indexOf` instead of `includes`?

Reply:
No reason, I completely forgot about includes. Replacing as it's nicer with includes. Thx!